PurchasesTests/Mocks/MockOfferingsFactory.swift

```diff
@@ -19,7 +19,7 @@ class MockOfferingsFactory: RCOfferingsFactory {
         }
         return Purchases.Offerings(
             offerings: [
-                "base": Purchases.Offering(
+                "base": Offering(
```

Comment:
Perhaps we could update the sample app as well?

Reply:
✅